lib/nexpose/common.rb

```diff
@@ -120,23 +120,89 @@ class Schedule
     attr_accessor :start_month
     attr_accessor :timezone
     attr_accessor :next_run_time
-    attr_accessor :template
 
-    def initialize(type, interval, start, enabled = true)
+    # scan-schedule attributes
+    attr_accessor :repeater_type
+    attr_accessor :scan_template_id
+
+    # @param [Time] start
+    def initialize(type, interval, start, enabled = true, scan_template_id = 'full-audit-without-web-spider')
```

Comment:
This will lead to incorrect behaviour as the default template for a schedule should match the template on the site to which it is being added.